apps/meteor/server/models/raw/LivechatDepartmentAgents.ts

```diff
@@ -113,4 +150,208 @@ export class LivechatDepartmentAgentsRaw extends BaseRaw<ILivechatDepartmentAgen
 	removeByDepartmentId(departmentId: string): Promise<DeleteResult> {
 		return this.deleteOne({ departmentId });
 	}
+
+	findByDepartmentId(departmentId: string): FindCursor<ILivechatDepartmentAgents> {
+		return this.find({ departmentId });
+	}
+
+	findOneByAgentIdAndDepartmentId(agentId: string, departmentId: string): Promise<ILivechatDepartmentAgents | null> {
+		return this.findOne({ agentId, departmentId });
+	}
+
+	saveAgent(agent: {
+		agentId: string;
+		departmentId: string;
+		username: string;
+		departmentEnabled: boolean;
+		count: number;
+		order: number;
+	}): Promise<Document | UpdateResult> {
+		return this.updateOne(
+			{
+				agentId: agent.agentId,
+				departmentId: agent.departmentId,
+			},
+			{
+				$set: {
+					username: agent.username,
+					departmentEnabled: agent.departmentEnabled,
+					count: parseInt(`${agent.count}`),
```

Comment:
```suggestion
					count: parseInt(`${agent.count}`),
					order: parseInt(`${agent.order}`),
```
Aren't they already numbers now?